Assets/Scripts/Dialogue/Core/Actions.cs

```diff
@@ -213,6 +230,44 @@ public class Actions : NodeCache {
             }
         }
 
+        //helper function for all sound-related funcions
+        //load all sound banks and help access the functions in AudioController.cs
+        private static AudioController LoadAudioController() {
```

Comment:
Please see my comments on AudioController and adjust these functions accordingly.